tests/integration/test_parallel_inception.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import print_function, unicode_literals
+
+import textwrap
+
+from tox.util.main import MAIN_FILE
+
+
+def test_parallel_inception(tmpdir, cmd):
+    # the outer config just has one env: graham
+    content = textwrap.dedent(
```

Comment:
I think you can transform this to initproj; which would make the file structure declaration a simple dictionary (no textrap + write_text) would be needed

Reply:
I'll try.

Reply:
done